src/Pagerfanta/View/Template/Template.php

```diff
@@ -63,4 +63,9 @@ protected function option($name)
 
         return $this->options[$name];
     }
+
+    protected function escape($value)
+    {
+        return htmlspecialchars($value, ENT_QUOTES, 'UTF-8');
```

Comment:
Adding the ENT_SUBSITITUTE flag for PHP 5.4+ for a better output on invalid UTF-8: https://github.com/symfony/symfony/blob/6b025418809a76371d44c8afe5d09c46875789cb/src/Symfony/Component/Debug/ExceptionHandler.php#L441